src/cpu/dyn_cache.h

```diff
@@ -783,15 +783,17 @@ static inline void dyn_mem_write(MAYBE_UNUSED void *ptr, MAYBE_UNUSED size_t siz
 
 static inline void dyn_cache_invalidate(MAYBE_UNUSED void *ptr, MAYBE_UNUSED size_t size)
 {
-#if defined(HAVE_SYS_ICACHE_INVALIDATE)
+#if defined(HAVE_BUILTIN_CLEAR_CACHE)
+	__builtin___clear_cache(static_cast<char *>(ptr), reinterpret_cast<char *>(reinterpret_cast<uintptr_t>(ptr) + size));		
```

Comment:
Suggest running for formatter.. it should nicely wrap this.

Reply:
yeah, the formatter screwed things up (due to platform defines I guess), I can run it and manually tweak the things it breaks.

Reply:
Argh.. darn.  Sounds good; whatever's easier!